cryptoconditions/types/ed25519.py

```diff
@@ -66,7 +66,7 @@ def sign(self, message, private_key):
         #   .update(Buffer.concat([this.messagePrefix, this.message]))
         #   .digest()
 
-        self.signature = sk.sign(message, encoding=None)
+        self.signature = sk.sign(message, encoding='bytes')
```

Comment:
make sure it is in line with the [cc spec](https://github.com/interledger/five-bells-condition/blob/master/src/types/ed25519.js#L82). Buffer means bytes